### PR TITLE
factory_bot

### PR DESCRIPTION
**Pull request by sivvoy**

### CODE REVIEW

File: spec/features/check_pdf_download_link_spec.rb [hunk 1]
```diff
@@ -43,19 +43,34 @@ def show
...
+    answer_earnings_and_benefits
+    answer_defend_claim_question
+    answer_representative
+    answer_disability_question
+    answer_employers_contract_claim
+    answer_additional_information
+    confirmation_of_supplied_details
 
-    expect(form_submission_page).to have_invalid_pdf_download
+    expect(form_submission_page).to have_valid_pdf_download
```
Comment (garytaylor) [inline spec/features/check_pdf_download_link_spec.rb]: Why has this changed to the opposite ?

File: spec/features/check_pdf_download_link_spec.rb [hunk 2]
```diff
@@ -43,19 +43,34 @@ def show
...
+    answer_earnings_and_benefits
+    answer_defend_claim_question
+    answer_representative
+    answer_disability_question
+    answer_employers_contract_claim
+    answer_additional_information
+    confirmation_of_supplied_details
 
-    expect(form_submission_page).to have_invalid_pdf_download
     expect(form_submission_page).to have_valid_pdf_download
```
Comment (garytaylor) [inline spec/features/check_pdf_download_link_spec.rb]: Why has this changed ?  it seems to be nothing to do with the factory bot stuff.  My guess is that this test was valid in that it needs to see it be invalid and then change to valid

File: spec/features/fill_in_confirmation_of_supplied_details_page_spec.rb [hunk 3]
```diff
@@ -8,21 +8,37 @@
...
+    answer_employers_contract_claim
+    answer_additional_information
+    confirmation_of_supplied_details
 
     expect(form_submission_page).to be_displayed
   end
 
   scenario "incorrectly will provide errors" do
-    given_i_am(:erroneously_entering_data_on_confirmation_of_supplied_details_page)
+    @claimant = FactoryBot.create(:claimant, :claimant_valid)
```
Comment (garytaylor) [inline spec/features/fill_in_confirmation_of_supplied_details_page_spec.rb]: One for @Stivaros - if we can change the use of instance vars to using standard rspec let's

File: test_common/page_objects/claimants_details_page.rb [hunk 4]
```diff
@@ -6,146 +6,179 @@ class ClaimantsDetailsPage < BasePage
...
+          element :selector, :css, 'input[type="radio"]'
+          def set(*args); selector.set(*args); end
+        end
+        element :no, :gds_multiple_choice_option, 'questions.agree_with_claimants_description_of_job_or_title.no.label' do
+          element :selector, :css, 'input[type="radio"]'
+          def set(*args); selector.set(*args); end
+        end
         section :disagree_claimants_job_or_title, :textarea_labelled, 'questions.agree_with_claimants_description_of_job_or_title.disagree_claimants_job_or_title.label', exact: false do
-          delegate :set, to: :root_element
+          def set(*args); root_element.set(*args); end
```
Comment (garytaylor) [inline test_common/page_objects/claimants_details_page.rb]: Why this change ?  The delegate methos is a standard way of doing this

File: spec/features/fill_in_confirmation_of_supplied_details_page_spec.rb (hunk 3 above)
Comment (Stivaros) [reply spec/features/fill_in_confirmation_of_supplied_details_page_spec.rb]: Will not make this change following further discussion

File: test_common/page_objects/claimants_details_page.rb (hunk 4 above)
Comment (Stivaros) [reply test_common/page_objects/claimants_details_page.rb]: As @sivvoy has explained this is due to an issue with site_prism.

File: spec/features/check_pdf_download_link_spec.rb (hunk 1 above)
Comment (Stivaros) [reply spec/features/check_pdf_download_link_spec.rb]: Fixed in e324b47

File: spec/features/check_pdf_download_link_spec.rb (hunk 2 above)
Comment (Stivaros) [reply spec/features/check_pdf_download_link_spec.rb]: Fixed in e324b47
